package.json

```diff
@@ -41,12 +41,11 @@
     "express-winston": "^2.0.0",
     "helmet": "^3.1.0",
     "images-require-hook": "^1.0.3",
+    "immutable": "^3.8.1",
     "isomorphic-fetch": "^2.2.1",
     "js-yaml": "^3.7.0",
     "memcached": "^2.2.2",
     "moment": "^2.17.1",
-    "nconf": "^0.8.4",
```

Comment:
I decided to drop nconf primarily because of the unavoidable dependency on the Node fs module. Even if we were to retain it for use in server-only code, we would need to code our own implementation for universal code. I think there are benefits for making the implementation as similar as possible throughout the codebase.